client/web/src/api.ts

```diff
@@ -239,6 +241,28 @@ export function useAPI() {
             .catch(handlePostError("Failed to update exit node"))
         }
 
+        /**
+         * "patch-serve-item" handles adding or updating an item in the
+         * node's serve config.
+         */
+        case "patch-serve-item": {
+          // todo: report metric?
```

Comment:
Are these todos awaiting input from Sam?

Reply:
More of a TODO for me - I have to look back at Sam's original metrics doc, I think he already had one listed out on there.